algorithms/c++/binary_search.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include<conio.h>
+#include<stdlib.h>
+
+#define MAX_SIZE 5
+
+using namespace std;
+
+int main()
+{
+    int arr_search[MAX_SIZE], i, element;
+    int f = 0, r = MAX_SIZE, mid;
```

Comment:
Codacy found an issue: [The scope of the variable 'mid' can be reduced.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6276107)